src/main/java/org/team1540/robot2023/commands/arm/CalculationTesting.java

```diff
@@ -0,0 +1,22 @@
+package org.team1540.robot2023.commands.arm;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import org.team1540.lib.math.Conversions;
+
+public class CalculationTesting {
```

Comment:
Ok, this really is not how this should be set up. It makes enough sense for testing, but it should not be part of the committed project